functions/lib/integration/parsers/order-to-tiny/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = (order, appData, storeId) => {
 
   const tinyOrder = {
     numero_pedido_ecommerce: orderRef,
-    data_pedido: formatDate(new Date(order.opened_at || order.created_at)),
+    data_pedido: formatDate(new Date(order.created_at || order.opened_at)),
```

Comment:
> De forma geral, idealmente é melhor enviar o pedido com a data de criação dele e não com a data em que foi aberto.

Por que? Se o pedido for aberto com data retroativa por que deveria enviar ele pro Tiny com a data atual? Quer dizer, se o cliente salvar hoje um pedido que foi feito mês passado a data correta do pedido é do mês passado, não hoje...

> Alguns clientes afirmam que a data de criação do pedido no tiny aparece sempre um dia depois do que foi de fato gerado.

E por que você acha que isso é por causa do `order.opened_at` 🤔 ?

Reply:
Até então eu acho que isso aqui só vai adicionar um novo bug, sem resolver o que você mencionou no issue 🙃

Reply:
No caso, o created_at está atualizando ainda? Pensei que ele era fixo da criação do pedido. De toda forma, testei depois e não faz muito sentido. Vou ter que testar uns problemas que estão ocorrendo nos pedidos e colocar log pra ver o que está rolando

Reply:
Quando edita o pedido não é alterado, seta `created_at` no POST e PUT, no admin usa PATCH pra atualizar pedido.
O ponto é que `created_at` é a data da criação/overwrite do documento na API, que não necessariamente é igual à data de criação do pedido.
Não sei se tem essa feature no admin, mas um usuário administrador pode criar na API um pedido retroativo, feito no passado, nesse caso o `created_at` não será a data de criação do pedido, o `opened_at` existe por isso, ele é que é responsável por ser a data do pedido independente das datas de manutenção do documento na API.